workflow-templates/release-branch.yml

```
@@ -0,0 +1,23 @@
+# Release branch starter workflow
+# See SHIP-0038: https://github.com/shipwright-io/community/pull/194
+name: Create Release Branch
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        required: true
+        type: string
+        description: "Semantic version for the release branch (vX.Y format)"
+      git-ref:
+        required: false
```

I think this is still required = it is not allowed to clear the field.

```suggestion
        required: true
```

---

I see how it works in the doc - yes, we'll want this to be required (and a supplied default of `main`)

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_dispatchinputs